independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/UpdateProjectCommandHandler.java

```diff
@@ -122,7 +122,11 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
 
     private static ArtifactCoords getProjectQuarkusPlatformBOM(ProjectState currentState) {
         for (ArtifactCoords c : currentState.getPlatformBoms()) {
-            if (c.getArtifactId().equals(ToolsConstants.DEFAULT_PLATFORM_BOM_ARTIFACT_ID)) {
+            if ((c.getGroupId().equals(ToolsConstants.IO_QUARKUS)
```

Comment:
Was there a reason for `io.quarkus` groupId check? RHBQ has a different groupId, for example.

Reply:
Ah right. I wanted to be more strict but maybe that's not a good idea. I will drop it.

Reply:
I removed the groupId check.